zap/src/main/java/org/zaproxy/zap/spider/parser/SpiderHtmlParser.java

```diff
@@ -270,25 +295,64 @@ private boolean parseSource(HttpMessage message, Source source, int depth, Strin
                             message, depth, baseURL, el, "srcset", this::srcSetProcessor);
         }
 
+        // Process IMPORT elements
+        elements = source.getAllElements(IMPORT_TAG);
+        for (Element el : elements) {
+            resourcesfound |=
+                    processAttributeElement(message, depth, baseURL, el, "implementation");
+        }
+
+        // Process content of container tags which hold text
+        String baseUrlForText = baseURL;
+        for (String tag : elementsWithText) {
+            elements = source.getAllElements(tag);
+            for (Element el : elements) {
+                Source s = new Source(el.getContent());
+                Matcher matcher = INLINE_CONTENT_URL_PATTERN.matcher(s.toString());
```

Comment:
Should just do `el.getContent().toString()`

Reply:
In the matcher, no need to intermediate my store in s

Reply:
Changed.

Reply:
👍